locale/en.json

```diff
@@ -142,6 +142,7 @@
   "CURATION_SKIPPED_MESSAGE": "Curation skipped - no images to curate.",
   "IDENTIFICATION_SKIPPED_MESSAGE": "Identification skipped.",
   "IDENTIFICATION_SKIPPED_NO_IMAGES_MESSAGE": "Identification skipped - no images to match.",
+  "IDENTIFICATION_SKIPPED_MIGRATED_MESSAGE": "Identification skipped - migrated sighting.",
```

Comment:
How about "Identification skipped - this sighting was migrated from the Zebra Wildbook" ?

Reply:
Is there an instance ID that we have access to that we could use to know that this instance is zebra?

Reply:
Ahh, no. How about "Identification skipped - this sighting was migrated from a Wildbook database."